src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,34 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+Port forwards activated on any `okteto up` command, independently of the dev container name.
+
+As in [forward](#forward-string-optional) in dev section, several formats are accepted for this, but it is necessary to indicate the name of the service to be forwarded using the `localPort:remoteService:remotePort` notation.
```

Comment:
I find this confusing. I think we can split this in two paragraphs: one for saying that this uses the same format of the other forward, and a second to talk about the service format?